spec/enumerable_methods_spec.rb

```diff
@@ -0,0 +1,87 @@
+require './lib/enumerable.rb'
+
+RSpec.describe Enumerable do
+  let(:arr) { [1, 2, 3, 8, 7, 4, 5, 6] }
+
+  describe '#my_any?' do
+    it 'Returns boolean if condition is passed' do
+      expect(arr.my_any? { |i| i >= 5 }).to eq(true)
+      expect(arr.my_any? { |i| i > 8 }).to eq(false)
+      expect(arr.my_any? { |i| i }).to eq(true)
+      expect(arr.my_any? { |i| i == 0 }).to eq(false)
+      expect(arr.my_any? { |i| i < 2 }).to eq(true)
+    end
+  end
+
+  describe '#my_none?' do
+    it 'Returns boolean if condition is passed' do
+      expect(arr.my_none? { |i| i >= 5 }).to eq(false)
+      expect(arr.my_none? { |i| i > 8 }).to eq(true)
+      expect(arr.my_none? { |i| i }).to eq(true)
+      expect(arr.my_none? { |i| i == 0 }).to eq(true)
+      expect(arr.my_none? { |i| i < 2 }).to eq(false)
+    end
+  end
+end
+
+RSpec.describe Enumerable do
```

Comment:
No need to repeat this line, you can encapsulate all the tests within first one

Reply:
We tried using that approach first, the rubocop will complain about too many lines, that's why we divide it